macros/models/_sector/transfers/eth/transfers_eth.sql

```diff
@@ -0,0 +1,71 @@
+{% macro transfers_eth(blockchain, eth_placeholder_contract, base_traces, base_transactions, erc20_transfer ) %}
+
+with eth_transfers as (
+    select
+        r."from"
+        ,r.to
+        --Using the ETH deposit placeholder address to match with prices tables
+        ,{{eth_placeholder_contract}} as contract_address
+        ,cast(r.value as double) AS value
+        ,cast(r.value as double)/1e18 as value_decimal
+        ,r.tx_hash
+        ,r.trace_address
+        ,r.block_time as tx_block_time
+        ,r.block_number as tx_block_number
+        ,substring(to_hex(t.data), 1, 10) as tx_method_id
+        ,cast(r.tx_hash as varchar) || '-' || array_join(r.trace_address,',') as unique_transfer_id
```

Comment:
Do we still want this field, or rather use the actual fields as our unique keys?